Week1/homework/queries_db_tb.js

```diff
@@ -0,0 +1,81 @@
+    "INSERT INTO country VALUES ('NLD','Netherlands','Europe','Western Europe',41526.00,1581,15864000,78.3,371362.00,360478.00,'Nederland','Constitutional Monarchy','Beatrix','AMSTERDAM')",
+    "INSERT INTO country VALUES ('ANT','Netherlands Antilles','North America','Caribbean',800.00,NULL,217000,74.7,1941.00,NULL,'Nederlandse Antillen','Nonmetropolitan Territory of The Netherlands','Beatrix','WILLEMSTAD')",
+    "INSERT INTO country VALUES ('BVT','Bouvet Island','Antarctica','Antarctica',59.00,NULL,0,NULL,0.00,NULL,'Bouvetoya','Dependent Territory of Norway','Harald V','')",
+    "INSERT INTO country VALUES ('CCK','Cocos (Keeling) Islands','Oceania','Australia and New Zealand',14.00,NULL,600,NULL,0.00,NULL,'Cocos (Keeling) Islands','Territory of Australia','Elisabeth II','WEST ISLAND')",
+    "INSERT INTO country VALUES ('ALB','Albania','Europe','Southern Europe',28748.00,1912,3401200,71.6,3205.00,2500.00,'Shqip ria','Republic','Rexhep Mejdani','TIRANA')",
+    "INSERT INTO country VALUES ('AND','Andorra','Europe','Southern Europe',468.00,1278,78000,83.5,1630.00,NULL,'Andorra','Parliamentary Coprincipality','','ANDORRA LA VELLA')",
+    "INSERT INTO country VALUES ('AUT','Austria','Europe','Western Europe',83859.00,1918,8091800,77.7,211860.00,206025.00,'osterreich','Federal Republic','Thomas Klestil','VIENNA')",
+    "INSERT INTO country VALUES ('RUS','Russian Federation','Europe','Eastern Europe',17075400.00,1991,146934000,67.2,276608.00,442989.00,'Rossija','Federal Republic','Vladimir Putin','MOSCOW')",
+    "INSERT INTO country VALUES ('ATA','Antarctica','Antarctica','Antarctica',13120000.00,NULL,0,NULL,0.00,NULL,'','Co-administrated','','')",
+    "INSERT INTO country VALUES ('CAN','Canada','North America','North America',9970610.00,1867,31147000,79.4,598862.00,625626.00,'Canada','Constitutional Monarchy, Federation','Elisabeth II','OTTAWA')",
+    "INSERT INTO country VALUES ('CHN','China','Asia','Eastern Asia',9572900.00,-1523,1277558000,71.4,982268.00,917719.00,'Zhongquo','Peoples Republic','Jiang Zemin','BEIJING')",
+    "INSERT INTO country VALUES ('BRA','Brazil','South America','South America',8547403.00,1822,170115000,62.9,776739.00,804108.00,'Brasil','Federal Republic','Fernando Henrique Cardoso','BRASILIA')",
+    "INSERT INTO country VALUES ('IND','India','Asia','Southern and Central Asia',3287263.00,1947,1013662000,62.5,447114.00,430572.00,'Bharat/India','Federal Republic','Kocheril Raman Narayanan','NEW DELHI')",
+    "INSERT INTO country VALUES ('ARG','Argentina','South America','South America',2780400.00,1816,37032000,75.1,340238.00,323310.00,'Argentina','Federal Republic','Fernando de la Rúa','BUENOS AIRES')",
+    "INSERT INTO country VALUES ('KAZ','Kazakstan','Asia','Southern and Central Asia',2724900.00,1991,16223000,63.2,24375.00,23383.00,'Qazaqstan','Republic','Nursultan Nazarbajev','NUR-SULTAN')",
+];
+
+const insertCities = [
+    "INSERT INTO city VALUES (ID,'Amsterdam','NLD','Noord-Holland',731200)",
```

Comment:
Yes it works and you even used the Auto Increment.
An other solution would have been to use the columns names where you insert the data :
INSERT table_name (col1,col2,..) VALUES (c1,c2,...). This way you can select which values you want to insert and you can use also Auto Increment or default Values.